appveyor.yml

```diff
@@ -11,52 +12,48 @@ configuration: Release
 test: OFF
 
 install:
-  - '"C:\Program Files\Microsoft SDKs\Windows\v7.1\Bin\SetEnv.cmd" /x86'
+  - call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvars32.bat"
 
   # sent environment variables for wxWidgets
-  - set WXWIN=C:\wxWidgets-3.0.2
+  - set WXWIN=C:\wxWidgets-3.1.2
   - set wxWidgets_ROOT_DIR=%WXWIN%
   - set wxWidgets_LIB_DIR=%WXWIN%\lib\vc_dll
   - cmd: SET PATH=%PATH%;%WXWIN%;%wxWidgets_LIB_DIR%;C:\Program Files (x86)\Poedit\Gettexttools\bin;
 
   # install dependencies:
-  - choco install poedit nsis -x86
+  - choco install poedit
+  
+  - ps: Start-FileDownload http://opencpn.navnux.org/build_deps/nsis-3.04-setup.exe
+  - cmd: nsis-3.04-setup.exe /S
 
   # Download and unzip wxwidgets
-  - ps: Start-FileDownload http://downloads.sourceforge.net/project/wxwindows/3.0.2/wxWidgets-3.0.2.zip
-  - cmd: 7z x wxwidgets-3.0.2.zip -o%WXWIN% > null
-
-  # some debugging information
-  - set
-  # - cmake --help
-
-  # build wxWidgets
-  - cmd: cd %WXWIN%\build\msw\
-  - cmd: nmake -f makefile.vc BUILD=release SHARED=1 CFLAGS=/D_USING_V120_SDK71_ CXXFLAGS=/D_USING_V120_SDK71_
-  - cmd: nmake -f makefile.vc BUILD=debug SHARED=1 CFLAGS=/D_USING_V120_SDK71_ CXXFLAGS=/D_USING_V120_SDK71_
+ - ps: Start-FileDownload http://opencpn.navnux.org/build_deps/wxWidgets-3.1.2.7z
```

Comment:
Does the second space make a difference?

Reply:
https://ci.appveyor.com/project/seandepagnier/climatology-pi/builds/22454391

```
Error parsing appveyor.yml: (Line: 30, Col: 2, Idx: 752) - (Line: 30, Col: 2, Idx: 752): While parsing a block mapping, did not find expected key.
```

Reply:
You are missing a space at the beginning of line 30. Use for example http://www.yamllint.com

Reply:
I thought that might be a problem. Thanks.